packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2076,18 +2086,33 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * sets up reconnection logic
+   * sets up listner for mercury event
    * @returns {undefined}
    * @public
    * @memberof Meeting
-   * @todo remove
-   * // TODO: remove??
    */
-  setReconnectListener() {
+  setMercuryListener() {
     // Client will have a socket manager and handle reconnecting to mercury, when we reconnect to mercury
     // if the meeting has active peer connections, it should try to reconnect.
     this.webex.internal.mercury.on(ONLINE, () => {
-      LoggerProxy.logger.log('Meeting:index#setReconnectListener --> Web socket online');
+      LoggerProxy.logger.info('Meeting:index#setMercuryListener --> Web socket online');
+
+      // Only send restore event when it was disconnected before and for connected later
+      if (!this.hasWebsocketConnected) {
+        Metrics.postEvent({
+          event: eventType.MERCURY_CONNECTION_RESTORED,
+          meeting: this
+        });
+        this.hasWebsocketConnected = true;
```

Comment:
Move this out of the if block it is in, so that we can set it to true even when we haven't connected before.

Reply:
ok i thought it will anyways goes in if its not connected so it will change it